conftest.py

```diff
@@ -73,6 +116,63 @@ def user_2() -> User:
     return user2
 
 
+@pytest.fixture
+def valid_user_data():
+    student_data = {'first_name': DEFAULT_FIRST_NAME,
+                    'last_name': DEFAULT_LAST_NAME,
+                    'email': DEFAULT_VALID_MAIL,
+                    'role': DEFAULT_USER_ROLE,
+                    'username': DEFAULT_VALID_USERNAME,
+                    'password1': DEFAULT_VALID_PASSWORD,
+                    'password2': DEFAULT_VALID_PASSWORD}
```

Comment:
Because it appears in another team code (code that has been approved), I assumed it was a good practice @ormergi.

Reply:
`User` does not have password1 and password2 fields for example.

Do you mean using the RegisterForm fields @ormergi ?

Reply:
> Because it appears in another team code (code that has been approved), I assumed it was a good practice @ormergi.

Well, what other projects do is not always the best practice :slightly_smiling_face:

> `User` does not have password1 and password2 fields for example.
> Do you mean using the RegisterForm fields @ormergi ?

I meant to use the `User` class instead of a dictionary, but as we saw together the problem is that `RegisterForm`
expects a dict on its `data` argument instead of a class type, what makes it harder to test.
We can workaround this by adding method that converts `User` to dict, and use it when passing the user data to the register form.

Also, note that`RegisterForm` inherits from `UserCreationForm` which already provides `username`, `password1` and `password2` attributes, `RegisterForm` should have add the rest of the attribute the app requires on registration.
Due to the fact that we dont have time to change `RegisterForm` before the final demo, we will have to go with
the current solution you preset on this PR.